src/import.js

```diff
@@ -250,6 +250,69 @@ padlock.import = (function(crypto, DisposableSource, Papa) {
         return data.split("\n")[0] == "url,username,password,extra,name,grouping,fav";
     }
 
+    /**
+     * Checks if a given string represents a KeePassX exported XML file
+     */
+    function isKeePassXML(xmldata) {
+        return xmldata.split("\n")[0] == "<!DOCTYPE KEEPASSX_DATABASE>";
```

Comment:
Why not just `xmldata.indexOf("<!DOCTYPE KEEPASSX_DATABASE>") !== -1`? Should be a little more reliable I think.